src/api/spec/README.md

```diff
@@ -187,14 +179,13 @@ You should only use shared examples where you have the exact same functionality
 Otherwise these tests get fast hard to refactor and review.
 In our experience, shared examples are used mainly for controllers. Since models are pretty different from each other, they (usually) do not share much logic.
 
-### Travis
-We use [travis-ci](https://travis-ci.org/) for continues integration.
+### Continuous Integration
+We use [travis-ci](https://travis-ci.org/) and [CircleCI](https://circleci.com) for continuous Integration.
 
 #### Setup
-As travis-ci runs on an Ubuntu machine, we need to add the OBS repository and install some OBS specific Ubuntu packages first.
-We do this in [dist/ci/obs_testsuite_travis_install.sh](https://github.com/openSUSE/open-build-service/blob/master/dist/ci/obs_testsuite_travis_install.sh).
-You can find the Ubuntu specific packages in this repository [http://download.opensuse.org/repositories/OBS:/Server:/Unstable/xUbuntu_12.04/](http://download.opensuse.org/repositories/OBS:/Server:/Unstable/xUbuntu_12.04/).
-We do not package the rubygems for Ubuntu, instead we use bundler to install them.
+In both services we use docker container for running our tests. The docker container are built with OBS in the container subprojects of [O:S:U](https://build.opensuse.org/project/subprojects/OBS:Server:Unstable) (e.g. https://build.opensuse.org/project/show/OBS:Server:Unstable:container:SLE12:SP3). With this approach we can test on our supported platforms like openSUSE or SLE and easily migrate to new platforms if necessary.
```

Comment:
I would add a `s` at the end of `we use docker container` and `The docker container`.